File: exercise/examples/fibonacci_recursion.py
print ("fibonacci sequence : 1 1 2 3 5 ...")
number = int (raw_input ("Enter the number:"))
def fibonacci (number):
    if (number == 1 or number == 2):
        return 1 
    else :
        return fibonacci(number-1)+fibonacci (number-2)
print (fibonacci (number))

File: exercise/examples/fibonacci_recursion.c
#include <stdio.h>
int fibonacci (int);
int main (void)
{
	printf ("Fibonacci sequences 1 1 2 3 5 8 .... \n");
	printf ("Please enter the term number : ");
	int number ;
	scanf("%d",&number);
	printf ("%d\n",fibonacci (number));
	return 0;



}
int fibonacci (int number)
{
	if (number == 1 || number == 2)
	
	{
		return 1;
	
	}
	else 
	{
		return fibonacci(number-1)+fibonacci(number-2);
	
	}


}
